persist/src/main/resources/db/migration/V1__Initial_Import.sql

```diff
@@ -39,7 +39,8 @@ CREATE TABLE IF NOT EXISTS teams
     team_name   VARCHAR(100)    NOT NULL,
     bio         TEXT            NOT NULL,
     finished    BOOLEAN         NOT NULL DEFAULT FALSE,
-    created_at  TIMESTAMP       NOT NULL DEFAULT CURRENT_TIMESTAMP
+    created_at  TIMESTAMP       NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    deleted_at  TIMESTAMP
```

Comment:
Fine for now but be careful changing these files

Reply:
Yeah, this was an exception since it should've been done before really but I'd missed it. All the tree/site db stuff will go in a second file